groups/prometheus/module-security-groups/main.tf

```diff
@@ -0,0 +1,42 @@
+# Security group for the Prometheus servers
+resource "aws_security_group" "prometheus_server" {
+  name        = "${var.tag_environment}-${var.tag_service}-server"
+  description = "Security group for Prometheus ${var.tag_environment} instances"
+  vpc_id      = var.vpc_id
+
+  ingress {
+    from_port   = 22
+    to_port     = 22
+    protocol    = "tcp"
+    cidr_blocks = var.ssh_cidrs
+    description = "SSH access"
+  }
+
+  ingress {
+    from_port   = 9090
+    to_port     = 9090
+    protocol    = "tcp"
+    cidr_blocks = var.prometheus_cidrs
```

Comment:
I am assuming only Grafana will be requiring access to Prometheus, it would be more secure to limit access based off the grafana security group rather than allowing access to an entire subnet.

Grafana security group can be pulled via a data block:
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/security_group

Reply:
Splitting this and addressing it in another story - DEVOP-1445